StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSheet.swift

```diff
@@ -123,7 +125,10 @@ public class CustomerSheet {
                              selectedPaymentMethodOption: selectedPaymentMethodOption,
                              merchantSupportedPaymentMethodTypes: merchantSupportedPaymentMethodTypes,
                              cbcEligible: elementsSession.cardBrandChoice?.eligible ?? false)
+                STPAnalyticsClient.sharedClient.logPaymentSheetEvent(event: .customerSheetLoadSucceeded,
+                                                                     duration: Date().timeIntervalSince(loadingStartDate))
             case .failure(let error):
+                STPAnalyticsClient.sharedClient.log(analytic: ErrorAnalytic(event: .customerSheetLoadFailed, error: error))
```

Comment:
1. Should we we be passing duration in for the failure as well?
2. Any reason why we chose to use `.logPaymentSheetEvent` for success but  `.log` for the failure case?

Reply:
No real reason to use log over logPaymentSheetEvent! I omitted duration here cause we don't really use it on the PaymentSheet event, but I don't think there is any harm in sending it incase we do end up using it and for consistency. 

Updated!